charts/brigade/values.yaml

```diff
@@ -44,6 +44,18 @@
 ## For full configuration, see the kashti chart in this repository and its values.yaml.
 kashti:
   enabled: true
+  kashti:
+    affinity: {}
```

Comment:
See note elsewhere re: preference to move the `affinity` section up a level (so, immediately under main `kashti`; we'd remove this second, nested `kashti`)